README.md

```diff
@@ -67,6 +67,7 @@ Environment variables can be used to enable or disable certain features of the S
 * `BOX` - the Vagrant base box to use. DEFAULT: `ubuntu/xenial64`
 * `ST2USER` - Username for st2. DEFAULT: st2admin
 * `ST2PASSWORD` - Password for st2. DEFAULT: `Ch@ngeMe`
+* `VERSION` - the version of StackStorm to install: `3.2`
```

Comment:
The version flag should be `3.2dev` for development installation while full semver `3.2.0` or `3.2.1` for stable.

See https://github.com/StackStorm/st2-packages/blob/f31ef5e9756ac7bd9f208f9dff65a8a5801ef0bb/scripts/st2_bootstrap.sh#L76-L78